<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastCheckerWidget</class>
 <widget class="QWidget" name="ContrastCheckerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="contrast">
       <property name="text">
        <string>Contrast: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="textColorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectTextColorButton">
       <property name="toolTip">
        <string>Select text color</string>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="backgroundColorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectBackgroundColorButton">
       <property name="toolTip">
        <string>Select background color</string>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="example"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="normalTextAA">
       <property name="text">
        <string>Normal Text AA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="normallTextAAA">
       <property name="text">
        <string>Normal Text AAA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="largeTextAA">
       <property name="text">
        <string>Large Text AA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="largeTextAAA">
       <property name="text">
        <string>Large Text AAA</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
